hunting-buddy.lic

```diff
@@ -43,7 +43,7 @@ class HuntingBuddy
 
   def need_boxes?
     return false unless @settings.box_loot_limit
```

Comment:
Should this check for `box_hunt_minimum`?

Reply:
Made the change.